frontend/src/app/main/ui/comments.scss

```diff
@@ -112,150 +106,147 @@
 }
 
 .replies {
+  @include bodySmallTypography;
   display: flex;
   gap: $s-8;
 }
 
-.total-replies {
+.replies-total {
   color: var(--color-foreground-secondary);
 }
-.new-replies {
+.replies-unread {
   color: var(--color-accent-primary);
 }
-// Thread-bubble
 
-.floating-thread-bubble {
-  @extend .comment-bubbles;
+.floating-preview-wrapper {
+  z-index: 1;
```

Comment:
Better to use an existing css variable instead of the number hard-coded

Reply:
Fixed